.storybook/YourTheme.ts

```diff
@@ -0,0 +1,41 @@
+// .storybook/YourTheme.ts
+import { create } from '@storybook/theming/create';
+import logo from '../public/assets/freenow-logo.svg';
+
+export default create({
+    base: 'light',
+
+    // Typography
+    fontBase: '"YOURNOWSans-Medium", "Open Sans", sans-serif',
+    fontCode: 'monospace',
+
+    brandTitle: 'FREENOW',
+    brandUrl: 'https://www.free-now.com/',
```

Comment:
This is the URL that the user is taken to when clicking on the FREENOW logo. IMO, it should be the URL of wave instead. What do you think?

Reply:
My way of thinking initially was that this is meant for brand URL as it's named, our lib being open-sourced would be a good way to link to company site, but I guess you're right, we can just use it as index page link. 🤔